recipes/loky/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "loky" %}
+{% set version = "2.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d042ce3600212f510770777f5f6166d9fd5fe86002851b202fdece3830b6e5b1
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - cloudpickle
+
+test:
+  imports:
+    - loky
+
+about:
+  home: https://github.com/tomMoral/loky
```

Comment:
I don't know the standard for `conda` packages but would it be better to link to the documentation (http://loky.readthedocs.io) for the home page?

Reply:
We use whatever is specified as the [`url` on PyPI]( https://github.com/tomMoral/loky/blob/2.1.4/setup.py#L60 ) for `home`. That said, have added the `doc_url` and `dev_url` fields. The former now points to the RTD page.